README.md

```diff
@@ -49,440 +49,6 @@ func main() {
 
 	fmt.Println(resp.Choices[0].Message.Content)
 }
-
-```
-
-### Other examples:
```

Comment:
Let's not remove examples from the README, as it's easy to reach

Reply:
Done!